server/src/main/java/io/crate/execution/ddl/tables/AddColumnTask.java

```diff
@@ -225,4 +237,11 @@ private static void mergeDeltaIntoExistingMapping(Map<String, Object> existingMa
             }
         );
     }
+
+    /**
+     * @return columns added by the last {@link #execute} call. Columns part of the {@link AddColumnRequest} which already existed are not included.
+     */
```

Comment:
As said, a new task will be created for each request, so there is no "last" execute call.

Reply:
There was a [comment](https://github.com/crate/crate/pull/14617#discussion_r1309808082) 
>And add a addedColumns.clear() in the start of .execute to make sure it could be called multiple times.
Currently not necessary given how it's used, but to match the docs. It's easier to explain this way than to say that it can only be used once.

I'm fine to get rid of clear and update docs to the prev version  - I don't think we will change the way we use it now (ie create task only once) and if we do we can always change code/docs.